sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/util/CosmosPagedFlux.java

```diff
@@ -84,4 +103,18 @@ public void subscribe(CoreSubscriber<? super T> coreSubscriber) {
             return Flux.fromIterable(elements);
         }).subscribe(coreSubscriber);
     }
+
+    private Flux<FeedResponse<T>> byPage(CosmosPagedFluxOptions pagedFluxOptions, Context context) {
```

Comment:
`CosmosPagedFlux` allows both paginated iteration and also item by item iteration.

I see that we covered the pageByPage iteration through adding `CosmosPagedFlux#byPage(.)` what about the item by item iteration shoudldn't we cover that as well?

Reply:
discussed offline.